pkg/controllers/disruption/emptiness.go

```diff
@@ -59,19 +65,32 @@ func (e *Emptiness) ComputeCommand(ctx context.Context, disruptionBudgetMapping
 
 	empty := make([]*Candidate, 0, len(candidates))
 	constrainedByBudgets := false
+	nodePoolDiffMapping := map[string]int{}
 	for _, candidate := range candidates {
 		if len(candidate.reschedulablePods) > 0 {
 			continue
 		}
-		if disruptionBudgetMapping[candidate.NodePool.Name] == 0 {
+
+		// We dont want to consider disruptionBudgets for scaling down static Nodeclaims
+		if candidate.NodePool.Spec.Replicas != nil {
```

Comment:
I think a lot of this emptiness code gets simplified if we don't handle the scale-down logic here